inst/extdata/hg19/ucsc_examplePEP/project_config.yaml

```diff
@@ -0,0 +1,7 @@
+metadata:
+        sample_table: samples_annotation.csv
+
+derived_attributes: [file_path]
+data_sources:
+        source1: LOLA/inst/extdata/hg19/regions/{sample_name}
```

Comment:
I'm a little confused about how this can work. You're pointing the path to something that's int he repository, but this has to be incorrect in the installed version because you're pointing to the `inst` folder, which is removed  upon install. so, I think this can't be right...

you need to test this and see if your examples work after installing the package.

Reply:
I"m guessing you should put the config file in the `/extdata` folder and then point with: `hg19/ucsc_example/regions/{sample_name}`

Reply:
@nsheff both the PEP folder and the actual regions are in the `hg19` directory so I think the path to the files should be `../ucsc_example/regions/{sample_name}`